Makefile

```diff
@@ -14,9 +14,10 @@
 
 # Image repo/tag to use all building/pushing image targets
 DOCKER_REGISTRY ?= ghcr.io/kubestellar/kubestellar
-IMAGE_TAG ?= 0.20.0-alpha.1
 CMD_NAME ?= controller-manager
-IMG ?= ${DOCKER_REGISTRY}/${CMD_NAME}:${IMAGE_TAG}
+IMAGE_TAG ?= $(shell git rev-parse --short HEAD) 
```

Comment:
I find it helpful to also include an indication of whether the working directory is a clean checkout or not, such as using the variable defined in https://github.com/kubestellar/kubestellar/blob/v0.15.0/Makefile#L21

Reply:
I find that less then useful but if you insist, I'll add it.

Reply:
I am not insisting.